crates/bevy_render/src/renderer/mod.rs

```diff
@@ -187,6 +187,10 @@ pub async fn initialize_renderer(
     let adapter_info = adapter.get_info();
     info!("{:?}", adapter_info);
 
+    if adapter_info.driver == "llvmpipe" {
+        warn!("The selected adapter is using the llvmpipe software renderer. This is likely to be very slow.");
```

Comment:
This should call out the "how to change your backend" docs or other troubleshooting advice.

Reply:
I suppose we don't have the how to change your backend docs anywhere? (ref #3406)